### ISSUE TITLE
Cannot choose gRCP method

### ISSUE THREAD

**Issue by Gibonachi** [kubeshop/tracetest#3278]
**Describe the bug**
I cannot choose a method when I try to create a new gRPC test

**To Reproduce**
Steps to reproduce the behaviour:
1. Click Create on the test list page 
2. Add new gRPC test, click next
3. Enter a name and click next
4. Choose a proto file
5. Open the 'Select method' dropdown

**Expected behaviour**
I would like to choose a method on the dropdown

**Screenshots**
![obraz](https://github.com/kubeshop/tracetest/assets/39117601/a8df2b77-9ad5-4799-bc85-926dba4ad255)

**Browser (please complete the following information):**
 - Browser: [Firefox] and [Chrome]

**Additional context**
Tracetest in the latest version updated 18.10.2023  
syntax = proto3
proto has one import - import "google/protobuf/timestamp.proto";
I'm not allowed to share a proto file. I hope you will be able to reproduce a bug. Proto works perfectly fine on Postman.

**Comment by adnanrahic** [comment]
Hey @Gibonachi ! Thanks for opening this issue. We'll take a look at this ASAP.

**Comment by mathnogueira** [comment]
Hey @Gibonachi , I'll check it out, as soon as I have found the root cause of that, I'll come back here and update this ticket and provide a PR to fix it.

**Comment by mathnogueira** [comment]
@Gibonachi Do you see any error in the browser console when you choose the grpc file? I'm trying to reproduce the issue but with no luck for now.  Does it rely on any other import? Maybe another local file?

**Comment by Gibonachi** [comment]
I see nothing interesting in the console or network tabs. 
Can you try on that proto (I changed the names of the data and their quantity, but I tried to maintain all the diverse structures and the overall format.): 

```
syntax = "proto3";

import "google/protobuf/timestamp.proto";

option csharp_namespace = "Something.Send";

service GrpcService {
  // Sends a greeting
  rpc SendStuff (SendRequest) returns (SendResponse);
}

message SendRequest {
  string customerId = 1;
  string somethingId = 2;
  google.protobuf.Timestamp startTimestamp = 3;
  google.protobuf.Timestamp endTimestamp = 4;
  repeated Sample positions = 5;
  optional string note = 6;
}

message Sample {
  google.protobuf.Timestamp timestamp = 1;
  double latitude = 2;
  optional double altitude = 3;
}

message SendResponse {}
```

**Comment by mathnogueira** [comment]
I'll give it a try, thanks for the example!

**Comment by mathnogueira** [comment]
With this proto I was able to reproduce the issue.

**Comment by jorgeepc** [comment]
Hi @Gibonachi we merged a PR with the fix. We are planning on doing a release later this week. I'll let you know when that is ready. Thanks for reporting this issue.

**Comment by jorgeepc** [comment]
Hey @Gibonachi we released `v0.14.5` yesterday. The fix for this issue should be there. Let us know if it's working for you. Thank you!
